### DISCUSSION TITLE
laravel 9 in not working

### DISCUSSION THREAD

**Discussion by umang200** [spatie/laravel-google-cloud-storage#40] [Q&A]
![image](https://user-images.githubusercontent.com/87923717/159247761-e9aa8101-735b-4df7-bd54-ac7d98c5e225.png)

**Comment by freekmurze** [comment]
Could you submit a PR that fixes this for you?

**Comment by j3j5** [comment]
We are having the same issue. Strangely enough, I've been able to reproduce it only in production (which actually caused us to rollback the upgrade). I've been trying to reproduce it locally but with no luck so far. If I can find any other info I'll post it here, hopefully with a PR, although I'm not sure whether the issue is here or on the [Flysystem driver itself](https://github.com/thephpleague/flysystem-google-cloud-storage).

**Comment by lalmat** [comment]
Hi Everyone,

We have a similar problem with laravel 9 where we can't put data to GCP. We can reproduce the problem on local machines, but we don't have any log to highlight it...

**Comment by lalmat** [comment]
Debbugger is our friend. Here is more informations :
<img width="858" alt="image" src="https://user-images.githubusercontent.com/2410437/173379272-de91760a-bcb5-4fc4-b243-cae591d55dc2.png">

Crash in vendor/laravel/framework/src/Illuminate/Filesystem/FilesystemAdapter.php @ line 344

We rolled back the bucket from Uniform Access To LCA and it's working now.
